spyder/py3compat.py

```diff
@@ -82,6 +82,16 @@
 #==============================================================================
 # Strings
 #==============================================================================
+def is_type_text_string(obj):

```

Comment:
What's the purpose of this function? I mean, why does it cover that it's not covered by `is_text_string`? (I guess the test will reveal it, and that's why I asked you for it :-).

Reply:
The propouse of this is to avoid letting in values in the `value_to_display` handling of strings. So for example, in the case of `Path` from the `path` package, it extends `str`, so a value passed to the `value_to_display` function currently would fall in managing a object `Path` as a string instead of making a default display of the element. For example, in Python 3, something that is instance of `str` normally should be type `str`, but if is something like `Path` it would be a instance of `str` and would have type `path.Path` as his type, then it shouldn't have to be treated as a string.

Reply:
This turned to be a good idea, thanks @dalthviz!